HW05.Rmd

```diff
@@ -0,0 +1,124 @@
+---
+title: "HW05"
+author: "Brock"
+date: "8/3/2020"
+output: html_document
+---
+#Goal: Tidy and join data :)
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+library("tidyverse") #Tidyverse functionalities
+library("knitr") #Nice outputs for tibbles/ dataframes
+library("here") #Pathing for importing data
+
+```
+
+Data importing is made easier with the here() function from the "here" package.
```

Comment:
Ooh this is a new package to me! Looking below, I see the benefit of "here" in specifying/locating file paths. I also used read_csv but not here(), but R was able to recognize the current folder I was in, so I only needed to specify the subfolders the files were in. For example, my code was: read_csv("CTRP_files/AUC_1.csv").